storage/flux/reader.go

```diff
@@ -593,19 +328,19 @@ READ:
 		done := make(chan struct{})
 		switch typedCur := cur.(type) {
 		case cursors.IntegerArrayCursor:
-			cols, defs := determineTableColsForGroup(gc.Keys(), flux.TInt)
+			cols, defs := determineTableColsForGroup(gc.Keys(), flux.TInt, gc.Aggregate(), key)
```

Comment:
`gc.Aggregate()` should always be `nil` here, correct?

Reply:
Because we're not pushing down grouped aggregates at the moment, yes.